catalog/conformance-tests/result-climatecamp-003.json

```diff
@@ -0,0 +1,15 @@
+{
+  "tested_solution": {
+    "solution_id": "climatecamp",
+    "version": "2.0.0"
+  },
+  "tested_by": {
+    "solution_id": "unilever",
+    "version": "1.0.0",
+    "solution_name": "Unilever"
```

Comment:
@StefanAmuraritei - Thanks for your response. This is a unique situation that we have now, since Unilever doesn't have a solution yet (I assume) that is already onboarded to the catalog, your test result may be not conclusive. We still would like to add your test result with Unilever to the Catalog, but have to figure out a way to do so without having a Unilever solution in the catalog. 
So I would request you to be kindly patient till we figure out the process to do so. This P will be open till then. We would definitely try to resolve this situation by early next week at the max. 

Appreciate you understanding and patience.

Cc: @zeitgeist @raimundo-henriques